paymentsheet/src/main/java/com/stripe/android/paymentsheet/IntentConfirmationInterceptorKtx.kt

```diff
@@ -27,8 +32,26 @@ internal suspend fun IntentConfirmationInterceptor.intercept(
                 requiresSaveOnConfirmation = paymentSelection.requiresSaveOnConfirmation,
             )
         }
+        null -> {
+            IntentConfirmationInterceptor.NextStep.Fail(
+                cause = IllegalStateException("Nothing selected."),
+                message = context.getString(R.string.stripe_something_went_wrong),
+            )
+        }
         else -> {
-            error("Attempting to confirm intent for invalid payment selection: $paymentSelection")
+            val exception =
+                IllegalStateException("Attempting to confirm intent for invalid payment selection: $paymentSelection")
+            val paymentSelectionName = paymentSelection.javaClass.name
```

Comment:
does this work if class names are obfuscated?

Reply:
It will not :( Good callout.